packages/account-v2/src/assets/connectedApps/ic-account-trash-can.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="115" height="87"><g fill="none"><rect width="108.14" height="60.3" x=".83" y="12.03" fill="#E0F4F3" rx="6"/><rect width="72.05" height="30.58" x="17.81" y="26.89" fill="#FFF" rx="5.09"/><path fill="#85ACB0" d="M36.68 38.8a1.71 1.71 0 00-2.33-.62l-1.82 1v-2.1a1.71 1.71 0 10-3.41 0v2.1l-1.82-1a1.702 1.702 0 10-1.7 2.95l1.82 1.05-1.82 1.06a1.688 1.688 0 00-.6 2.32c.304.53.868.859 1.48.86.298-.003.59-.082.85-.23l1.82-1v2.1a1.71 1.71 0 003.41 0v-2.1l1.82 1c.26.147.552.226.85.23a1.71 1.71 0 00.86-3.18l-1.82-1.06 1.82-1.05a1.71 1.71 0 00.59-2.33zm15.85 0a1.701 1.701 0 00-2.32-.62l-1.83 1v-2.1a1.7 1.7 0 10-3.4 0v2.1l-1.83-1a1.701 1.701 0 10-1.7 2.95l1.82 1.05-1.82 1.06a1.692 1.692 0 00-.63 2.32c.304.53.868.859 1.48.86.298-.003.59-.082.85-.23l1.83-1v2.1a1.71 1.71 0 003.41 0v-2.1l1.83 1c.259.15.552.228.85.23a1.711 1.711 0 00.85-3.18l-1.82-1.06 1.82-1.05a1.71 1.71 0 00.61-2.33zm15.85.01a1.692 1.692 0 00-2.32-.63l-1.83 1v-2.1a1.7 1.7 0 10-3.4 0v2.1l-1.83-1a1.701 1.701 0 10-1.7 2.95l1.82 1.05-1.82 1.06a1.71 1.71 0 00.85 3.18c.298-.002.591-.08.85-.23l1.83-1v2.1a1.7 1.7 0 103.4 0v-2.1l1.83 1c.259.15.552.228.85.23a1.711 1.711 0 00.85-3.18l-1.82-1.06 1.82-1.05a1.691 1.691 0 00.62-2.32zm15.86 0a1.71 1.71 0 00-2.33-.63l-1.82 1v-2.1a1.71 1.71 0 00-3.41 0v2.1l-1.83-1a1.702 1.702 0 10-1.7 2.95L75 42.19l-1.82 1a1.722 1.722 0 00-.024 2.986c.53.312 1.188.318 1.724.014l1.83-1v2.1a1.7 1.7 0 103.4 0v-2.1l1.83 1c.26.148.552.227.85.23a1.7 1.7 0 00.85-3.18l-1.82-1 1.82-1.06a1.689 1.689 0 00.6-2.37z"/><ellipse cx="91.99" cy="63.85" fill="#FF444F" rx="23.01" ry="23"/><path fill="#FFF" d="M84.26 71.72a3.54 3.54 0 003.53 3.54h8.14a3.54 3.54 0 003.53-3.54V58.51a1 1 0 00-1-1H85.25a1 1 0 00-1 1l.01 13.21zm15.47-18h-3a1.003 1.003 0 01-.71-.3l-.68-.68a.999.999 0 00-.71-.29H89.1a.999.999 0 00-.7.29l-.68.68c-.187.19-.443.299-.71.3H84a1 1 0 00-1 1v.54a1 1 0 001 1h15.74a1 1 0 001-1v-.54a1 1 0 00-1.01-1z"/><rect width="23.85" height="23.85" x="41.91" y=".11" fill="#E0F4F3" rx="11.92"/></g></svg>
```

Comment:
Any plan to move this to the quill icons repo?

Reply:
@utkarsha-deriv  has already asked for it in the channel. Once it is available, it can be replaced